test/e2e/packagemanifest_e2e_test.go

```diff
@@ -168,6 +169,74 @@ var _ = Describe("Package Manifest", func() {
 			require.True(GinkgoT(), ok, "Expect this image %s to exist in the related images list\n", v)
 		}
 	})
+	It("updating CatalogSource", func() {
```

Comment:
Could the test be more structured to use ginkgo's container blocks and replacing assertions with gomega matcher library? You could refer to #1424 as a reference

Reply:
you could refer to this [cheatsheet](https://docs.google.com/document/d/1SwDGGrOTpCfE32gPE8GyFOwyZ058BUMhcI_KkSzU7Bg/edit#)

Reply:
Right, that would be much better, but may I suggest that we do that in a separate PR and have this test arranged exactly as the other tests in this file? IMO refactoring should be its own PR rather than smuggles in bug fixes.

Reply:
I updated the test to at least not to use assert and conform to the other tests.

Reply:
This is a new test -- why create more future work for ourselves? If this bug were on fire, I can see the argument for addressing the test feedback in a follow-up, but it doesn't seem to be urgent enough to do that.

Reply:
Lets wait for https://github.com/operator-framework/operator-lifecycle-manager/pull/1439/files to merge first.

Reply:
I agree with @benluddy

Reply:
Yes, I agree, but I am saying waiting for https://github.com/operator-framework/operator-lifecycle-manager/pull/1439/files will resolve all of our concerns. That PR is very close to merging.

/hold